provider/cldr/src/transform/time_zones/convert.rs

```diff
@@ -97,18 +100,28 @@ impl From<TimeZoneNames> for ExemplarCitiesV1<'_> {
                             key.push('/');
                             key.push_str(&inner_key);
                             match place_or_region {
-                                LocationOrSubRegion::Location(place) => place
-                                    .exemplar_city()
-                                    .map(|city| vec![(key, city)])
-                                    .unwrap_or_default(),
+                                LocationOrSubRegion::Location(place) => {
+                                    match bcp47_tzid_data.get(&key) {
+                                        Some(bcp47) => place
+                                            .exemplar_city()
+                                            .map(|city| vec![(bcp47.clone(), city)])
+                                            .unwrap_or_default(),
+                                        None => Default::default(),
```

Comment:
Issue: If the mapping is not found, we should not fall back to `None`. I think we should probably return an error. For now you can panic.